.github/workflows/ci.yml

```diff
@@ -1087,6 +1087,104 @@ jobs:
           path: diagnostics*.gz
           retention-days: 1
 
+  # Runs tests on `tests/cli/partition_test`
+  partition_test:
+    name: Partitions Test (${{ matrix.test }}) on ${{ matrix.os }} for ${{ matrix.rust-target }}
+    runs-on: ${{ matrix.os }}
+    env:
+      FLUVIO_BIN: "~/bin/fluvio"
+      SERVER_LOG: fluvio=info
+    needs:
+      - build_primary_binaries
+      - config
+    strategy:
+      matrix:
+        os:
+          - ubuntu-latest
+        rust-target:
+          - x86_64-unknown-linux-musl
+        test:
+          - multiple_partitions
+    steps:
+      - name: Checkout Source Code
+        uses: actions/checkout@v3
+
+      # Download Artifacts from Development Build (This Commit)
+      - name: Download artifact - fluvio
+        uses: actions/download-artifact@v3
+        with:
+          name: fluvio-${{ matrix.rust-target }}
+          path: ~/bin
+
+      - name: Download artifact - fluvio-run
+        uses: actions/download-artifact@v3
+        with:
+          name: fluvio-run-${{ matrix.rust-target }}
+          path: ~/.fluvio/extensions
+
+      - name: Setup K8s Cluster
+        run: |
+          curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | TAG=${{ env.K3D_VERSION }} bash
+          ./k8-util/cluster/reset-k3d.sh
+
+      - name: Wait 15s for K3D Reset
+        run: sleep 15
+
+      # This will load into ~/.fluvio/bin which will be in PATH
+      - name: Install Stable CLI and start Fluvio cluster
+        run: |
```

Comment:
why this is necessary?   We only need to test same version

Reply:
Im setting up necessary Fluvio files and then Im replacing the binaries with the ones from the current PR as we do here:

https://github.com/infinyon/fluvio/blob/master/.github/workflows/ci.yml#L971-L983

Reply:
that's different tests.  it is designed to test compatibility between dev and release.  In this case, only need to test between dev